src/scss/_admonitions.scss

```diff
@@ -1,14 +1,20 @@
-// Admonitions CSS inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
+// Admonitions CSS originally inspired by https://squidfunk.github.io/mkdocs-material/getting-started/
+
+@function hexToRGB($hex) {
+  @return red($hex), green($hex), blue($hex);
+}
+
 .admonition {
   width: 95%;
   margin: 1.5625em auto;
-  padding: 0 .6rem .8rem .6rem !important;
+  padding: 0 0.6rem 0.8rem 0.6rem !important;
   overflow: hidden;
   page-break-inside: avoid;
-  border-left: .2rem solid $blue;
-  border-radius: .1rem;
-  box-shadow: 0 0.2rem 0.5rem rgba(0,0,0,.05), 0 0 0.05rem rgba(0,0,0,.1);
-  transition: color 250ms,background-color 250ms,border-color 250ms;
+  border-left: 0.2rem solid var(--color-admonition-primary);
```

Comment:
for some reason, this doesn't seem to be working with a variable (in the preview the admonitions have no left border (https://421-130237506-gh.circle-artifacts.com/0/html/demo/demo.html#admonitions), compare with master: https://pydata-sphinx-theme.readthedocs.io/en/latest/demo/demo.html#admonitions). 

Below, this color is also overriden with `border-color`, which in itself seems to be working, because if I remove the color from the above `border-left` in firefox web tools, the borders appear again.

Reply:
Ah, actually, I suppose this is because it's a wrong/unknown variable. `--color-admonition-primary` is not defined